tools/reviewer/job/sync/full_github_pr.proto

```diff
@@ -68,6 +52,9 @@ message ReviewComment {
   string body = 11;
   string created_at = 12;
   string updated_at = 13;
+  Links _links = 14;
+  // bool created_in_reviewer = 15;
```

Comment:
***oferb's comment:*
Should this be deleted?
`// bool created_in_reviewer = 15;`

Also below:
`// bool created_in_reviewer = 6;`